apps/aklite-offline/cmds.h

```diff
@@ -57,7 +64,11 @@ class CheckCmd : public Cmd {
 
 class InstallCmd : public Cmd {
  public:
-  InstallCmd() : Cmd("install", _options) {
+  InstallCmd()
+      : Cmd("install",
+            "Installs the selected target. If no target name is specified, the highest version in the bundle will "
```

Comment:
I think it should be without `s` endings, it not like `it installs` it is rather instruction what to do `do install`. The same for the rest commands. The `fioctl` defines verbs/actions without ending "(e)s" as well.

Reply:
I've now removed the "s"s from the desctiption.